locale/en/user.po

```diff
@@ -852,3 +852,18 @@ msgstr "This ORCID has not been verified. Please remove this unverified ORCID an
 
 msgid "user.removeRole.message"
 msgstr "Are you sure want remove this role permanently?"
+
+msgid "user.futureRole.notification.message"
+msgstr "Your role is scheduled to begin on "
+
+msgid "user.futureRole.notification.description"
+msgstr "Until then, you can review and update your profile. If you believe this is an error, please contact journal administrator."
```

Comment:
Should not explicitly reference journal since this will apply to OMP and OPS as well.